test/module-hooks/test-module-hooks-builtin-require.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const { registerHooks } = require('module');
+
+const schemelessBlockList = new Set([
+  'sea',
+  'sqlite',
+  'test',
+  'test/reporters',
+]);
+
+const testModules = [];
+for (const mod of schemelessBlockList) {
+  testModules.push(`node:${mod}`);
+}
+
+const hook = registerHooks({
+  resolve: common.mustCall((specifier, context, nextResolve) => nextResolve(specifier, context), testModules.length),
```

Comment:
Can you split the resolve tests into a different file?

Reply:
I think I can't, the bug appear only when there're both `resolve` and `load` hook, neither of them will pass the test